README.md

```diff
@@ -1179,6 +1180,26 @@ module.exports = {
 }
 ```
 
+#### Content Security Policy
+
+Next.js supports [CSP](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP). You can set your CSP Poliy in your Next.js configuration.
+
+```js
+// next.config.js
+module.exports = {
+  cspPolicy: "default-src 'self';"
+}
+```
+
+Node: In order to support dynamic styles nodejs uses a `nonce` for `style-src`. This is done automatically. If you use a different CSS-in-JS library, you can use the `cspNonce` document parameter to get the nonce set in the header. 
+
+If you use static exporting, you must do **one** of the following:
+
+1. set `unsafeCSPMeta: true` in `next.config.js` and use `style-src 'self' 'unsafe-inline'` in your policy.
```

Comment:
This seems to refer to the previous name 🤔

Reply:
Yeah sorry these HMR tests are failing for the most random reasons and I’m just pushing to test on travis instead of my machine...